scripts/utils.js

```diff
@@ -0,0 +1,35 @@
+require("dotenv").config()
+
+module.exports = {
+  getRealmNetworkFromArgs,
+  getRealmNetworkFromNetwork,
+}
+
+function getRealmNetworkFromArgs () {
+  let realm = process.env.WITNET_EVM_REALM
+    ? process.env.WITNET_EVM_REALM.toLowerCase()
+    : "default"
+  let network = "development"
+  const args = process.argv.join("=").split("=")
+  const networkIndex = args.indexOf("--network")
+  if (networkIndex >= 0) {
+    network = args[networkIndex + 1]
+    realm = (network || "default").split(".")[0]
+  }
+  if (realm === "ethereum") realm = "default"
+  return [realm, network]
+}
```

Comment:
Same comment as in https://github.com/witnet/witnet-solidity-bridge/pull/201#discussion_r717375059